### PR TITLE
Update templates to use includes

### PR DESCRIPTION
**Pull request by yammes08**
- Update all templates (except French guide templates) to use new includes feature.
    - Note: items where users are expected to make an inclusion or scoring choice are retained in the templates. All other items are moved to includes
- Fixed 10-bit custom format placement in anime templates to reflect user scoring choice status
- Changed include filename and ID for SQP-1 (2160p) quality profile choices
    - Note, as SQP-1 (2160p) features multiple quality profile configurations relating to `qualities`, multiple includes have been created to cover the options
- A little fix for Fab

### CODE REVIEW

**Review by fabricionaweb — APPROVED** [review]
Loved it

**Comment by rcdailey** [comment]
I plan to review this either today or this weekend. I want to give it the time it deserves.

File: radarr/templates/sqp/sqp-1-2160p.yml [hunk 1]
```diff
@@ -15,124 +15,35 @@ radarr:
...
+        # min_format_score: 10
+
+    include:
+    # Choose ONE of the following four options to determine quality profile behavior.
+      - template: sqp-1-2160p-qp-default  # Default
+      # - template: sqp-1-2160p-qp-imax-e  # Prefer IMAX-E, enable IMAX-E below
+      # - template: sqp-1-2160p-4k-only-qp-default  # Default, 4K qualities only
+      # - template: sqp-1-2160p-4k-only-qp-imax-e  # Prefer IMAX-E, 4K only, enable IMAX-E below
+      # Do not comment out the next line as it enables this profile's custom formats
+      - template: sqp-1-2160p-shared
```
Comment (rcdailey) [inline radarr/templates/sqp/sqp-1-2160p.yml:30]:
```suggestion
    # Choose ONE of the following four options to determine quality profile behavior.
      - template: sqp-1-2160p-qp-default  # Default
      # - template: sqp-1-2160p-qp-imax-e  # Prefer IMAX-E, enable IMAX-E below
      # - template: sqp-1-2160p-4k-only-qp-default  # Default, 4K qualities only
      # - template: sqp-1-2160p-4k-only-qp-imax-e  # Prefer IMAX-E, 4K only, enable IMAX-E below

      # Do not comment out the next line as it enables this profile's custom formats
      - template: sqp-1-2160p-shared
```

nitpick: I'd suggest introducing some vertical whitespace here to better separate things for the users.

File: radarr/templates/sqp/sqp-2.yml [hunk 2]
```diff
@@ -15,111 +15,25 @@ radarr:
...
-          - name: Remux-1080p
+    include:
+      - template: sqp-2
 
 # Custom Formats
 # This section creates and/or updates Custom Formats in your Radarr instance,
 # and applies them and their scores to Quality Profiles
 # Documentation: https://recyclarr.dev/wiki/yaml/config-reference/custom-formats/
+
+# You must comment out lines 27, 28, 49 and 50 if you disable all the custom formats below
```
Comment (rcdailey) [inline radarr/templates/sqp/sqp-2.yml:26]: This is actually a really interesting use case. I think I will update Recyclarr to allow empty `trash_ids` lists, in order to make the editing experience of the templates a little better.

**Review by rcdailey — APPROVED** [review]
yammes, you are an absolute unit. I am amazed that you got all this done. Really great work. Reviewing this was helpful for me, too. I can think of a couple of QoL changes I can make in Recyclarr that will help simplify even further later on.

Thanks for allowing me time to look at this for you. Even though I've left a few comments, I'm going to approve as they are completely optional. Feel free to merge with no further changes if you'd like. Fantastic job.

File: radarr/includes/uhd-bluray-web.yml [hunk 3]
```diff
@@ -63,16 +63,20 @@ custom_formats:
       - 9c38ebb7384dada637be8899efa68e6f  # SDR
 
       # Streaming Services
+      - cc5e51a9e85a6296ceefe097a77f12f4  # BCORE
+      - 16622a6911d1ab5d5b8b713d5b0036d4  # CRiT
+      - 2a6039655313bf5dab1e43523b62c374  # MA
+    quality_profiles:
+      - name: UHD Bluray + WEB
```
[the remark this answers is not in the document]
Comment (yammes08) [reply radarr/includes/uhd-bluray-web.yml:70]: Yes - although as I understand it we'd need to progress functionality to allow custom format block not directly assigned to a quality profile, right? Common CF 'blocks' (i.e., includes) such as Audio Formats or Streaming services would be useful for both includes and templates.

File: radarr/templates/sqp/sqp-2.yml (hunk 2 above)
Comment (yammes08) [reply radarr/templates/sqp/sqp-2.yml:26]: That would definitely clean up this area of the config, thanks!

**Pull request merged**